### ISSUE TITLE
Split Auto-Type option "match entry title & entry URL"

### ISSUE THREAD

**Issue by GUiHKX** [keepassxreboot/keepassxc#638]
KPXC now has this useful option to match URLs in titles (#556):

![kpxc](https://user-images.githubusercontent.com/626206/27087286-f8dcc34e-502a-11e7-93fc-6ce1115af1f6.png)

But I wish that option was split in two instead (like KeePass does):
```
[ ] An entry matches if its title is contained in the target window title
[x] An entry matches if its URL is contained in the target window title
```
I want this feature because I don't use the "match entry title" option, it causes some annoying conflicts for me.

Thanks.

**Comment by TheZ3ro** [comment]
I think there is already an issue for that but i can't find it. Anyway I was already planning on doing this

**Comment by GUiHKX** [comment]
Do you think this will be included in the v2.2.0 release? Thank you.

**Comment by TheZ3ro** [comment]
@GUiHKX no, probably I will add this in v2.2.1

**Comment by GUiHKX** [comment]
Hello, I've recently switched KPXC from git version to the stable one (v2.2.2) and I noticed this option is not there yet. Is it on purpose? Thanks.

**Comment by phoerious** [comment]
It's only been merged into 2.3.0 and has not been backported.

**Comment by GUiHKX** [comment]
I see, thanks for the clarification
